addons/crm/models/crm_lead.py

```diff
@@ -151,6 +151,9 @@ class Lead(models.Model):
     recurring_revenue_monthly = fields.Monetary('Expected MRR', currency_field='company_currency', store=True,
                                                compute="_compute_recurring_revenue_monthly",
                                                groups="crm.group_use_recurring_revenues")
+    recurring_revenue_prorated = fields.Monetary("Prorated Recurring Revenues", currency_field="company_currency",
+                                                 compute="_compute_recurring_revenue_prorated",
+                                                 groups="crm.group_use_recurring_revenues")
```

Comment:
As your field is slightly different from others (not stored), and to keep alphabetical ordering, I would not put it between the others, but below.